discord/wrappers.go

```diff
@@ -0,0 +1,33 @@
+package discord
+
+import (
+	"github.com/bwmarrin/discordgo"
+)
+
+type SessionWrapper struct {
+	session *discordgo.Session
+}
```

Comment:
One way to avoid these unwieldy wrappers is to use struct composition:
```type SessionWrapper struct {
	*discordgo.Session
}```
This way, the only new members will be the ones that were field accessors before. In this case, `GetState` will be the only remaining member.

On the other hand, I kind of like explicitness.

Reply:
Oh! Nevermind. The wrappers should all vend wrappers as well, so this nifty trick won't work.